Transmission.xcodeproj/project.pbxproj

```diff
@@ -2396,6 +2397,42 @@
 			shellPath = /bin/sh;
 			shellScript = "cd third-party/dht && rm -f dht && ln -s . dht\n";
 		};
+		EDF2BD6C273FF71F00929B94 /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "cd third-party/libnatpmp && rm -f VERSION\n";
```

Comment:
Just delete `VERSION` file before this target is built.
One downside - it makes subodule dirty.
But previous build schema makes submodules dirty anyway.